README.md

```diff
@@ -379,17 +411,103 @@ Documentation Forthcoming
 
 ### Modules
 
-#### `loadModuleData`
+#### Loading Modules
 
-Documentation Forthcoming
+In either the Server or the Browser, there are two methods to select and load specific Holocron Modules: 1) Use Routes defined in the [child routes configuration](#routing) to match a URL path to a Holocron Module. 2) Or use dispatch-able methods in the [Holocron API](https://github.com/americanexpress/holocron/blob/master/packages/holocron/API.md) to load Holocron Modules and render their contents with a React Component.
+
+Both methods are described in the following.
+
+**Contents**
+* Route Component
+  * [`ModuleRoute`](#moduleroute)
+* Load and Render
+  * [`RenderModule`](#rendermodule)
+  * [`holocronModule`](#holocronmodule)
+  * [`composeModules`](#composemodules)
+
+##### React Router Component
```

Comment:
This still says `React Router`. Can you update to say 'One App Router Component`?

Reply:
Fixed in 7f9463d